contracts/test-utils/src/log_utils.ts

```diff
@@ -32,3 +32,18 @@ export function verifyEvents<TEventArgs>(
         expect(log).to.deep.equal(expectedEvents[index]);
     });
 }
+
+/**
+ * Given a collection of logs, verifies that matching events are identical.
+ */
+export function verifyEventsFromLogs<TEventArgs>(
```

Comment:
Can we get rid of either this or `verifyEvents` now that the latter has been merged?

Reply:
Decided to just refactor `verifyEvents()` to use `verifyEventsFromLogs()` so they can coexist without duping code.